.github/workflows/pull_request.yml

```diff
@@ -3,35 +3,26 @@ name: Build Chain
 on: [pull_request]
 
 jobs:
-  build-chain-openjdk8:
+  build-chain:
+    strategy:
+      matrix:
+        java-version: [8, 11]
+      fail-fast: true
     runs-on: ubuntu-latest
-    name: Pull Request openjdk8
+    name: Maven Build
     steps:
-    - uses: actions/checkout@v2
-    - name: Build Chain
-      id: build-chain
-      uses: kiegroup/github-action-build-chain@openjdk8
-      with:
-        parent-dependencies: 'lienzo-core'
-        child-dependencies: 'droolsjbpm-build-bootstrap'
-        build-command: 'mvn -e -nsu -Dfull -Pwildfly clean install -Prun-code-coverage  -Dcontainer.profile=wildfly -Dcontainer=wildfly -Dintegration-tests=true -Dmaven.test.failure.ignore=true -DjvmArgs="-Xms1g -Xmx4g"'
-        build-command-upstream: "mvn -e -T1C clean install -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true"
-        workflow-file-name: "pull_request.yml"
-      env:
-          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-  build-chain-openjdk11:
-    runs-on: ubuntu-latest
-    name: Pull Request openjdk11
-    steps:
-    - uses: actions/checkout@v2
-    - name: Build Chain
-      id: build-chain
-      uses: kiegroup/github-action-build-chain@openjdk11
-      with:
-        parent-dependencies: 'lienzo-core'
-        child-dependencies: 'droolsjbpm-build-bootstrap'
-        build-command: 'mvn -e -nsu -Dfull -Pwildfly clean install -Prun-code-coverage  -Dcontainer.profile=wildfly -Dcontainer=wildfly -Dintegration-tests=true -Dmaven.test.failure.ignore=true -DjvmArgs="-Xms1g -Xmx4g"'
-        build-command-upstream: "mvn -e -T1C clean install -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true"
-        workflow-file-name: "pull_request.yml"
-      env:
-          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java-version }}
+      - name: Build Chain ${{ matrix.java-version }}
+        id: build-chain
+        uses: kiegroup/github-action-build-chain@v1.4
+        with:
+          parent-dependencies: 'lienzo-core'
+          child-dependencies: 'droolsjbpm-build-bootstrap'
+          build-command: 'mvn -e -nsu -Dfull -Pwildfly clean install -Prun-code-coverage  -Dcontainer.profile=wildfly -Dcontainer=wildfly -Dintegration-tests=true -Dmaven.test.failure.ignore=true -DjvmArgs="-Xms1g -Xmx4g"'
```

Comment:
please remove that -DjvmArgs in all pull_request.yml files

Reply:
Done